src/components/Certify/Certify.vue

```diff
@@ -0,0 +1,157 @@
+            :value="certifiedBy"
+            :rules="[(v) => !!v || 'A person\'s legal name is required.']"
+            @input="emitCertifiedBy($event)"
+          />
+        </div>
+      </div>
+      <v-checkbox
+        :value="isCertified"
+        @change="emitIsCertified($event)"
+        id="isCertified-checkbox"
+      >
+        <template slot="label">
+          <div class="certify-stmt">
+            I,
+            <strong>{{ trimmedCertifiedBy || "[Legal Name]" }}</strong>
+            , certify that I have relevant knowledge of the
+            {{ entityDisplay || "association" }} and that I am authorized to
+            make this filing.
+          </div>
+        </template>
+      </v-checkbox>
+      <p class="certify-clause">Date: {{ currentDate }}</p>
+      <p class="certify-clause">{{ message }}</p>
+    </div>
+  </v-card>
+</template>
+
+<script lang="ts">
+import { Component, Vue, Prop, Emit } from 'vue-property-decorator'
+import { mapState } from 'vuex'
+
+@Component({
+  computed: {
+  // ...mapState(['currentDate'])
+  }
+})
+export default class Certify extends Vue {
+
+  @Prop({ default: '' })
+  private currentDate!: string
+
+  /** Certified By prop. */
+  @Prop({ default: '' })
+  private certifiedBy: string
+
+  /** Is Certified prop. */
+  @Prop({ default: false })
+  private isCertified: boolean
+
+  /** Message prop. */
+  @Prop({ default: '' })
+  private message: string
+
+  /** Entity Display prop. */
+  @Prop({ default: '' })
+  private entityDisplay: string
+
+  /** Called when component is created. */
+  private created (): void {
+    // inform parent of initial validity
+    this.emitValid(Boolean(this.trimmedCertifiedBy && this.isCertified))
+  }
+
+  /** The trimmed "Certified By" string (may be ''). */
+  private get trimmedCertifiedBy (): string {
+    // remove repeated inline whitespace, and leading/trailing whitespace
+    return this.certifiedBy && this.certifiedBy.replace(/\s+/g, ' ').trim()
+  }
+
+  /** Emits an event to update the Certified By prop. */
+  @Emit('update:certifiedBy')
+  private emitCertifiedBy (certifiedBy: string): string {
+    // remove repeated inline whitespace, and leading/trailing whitespace
+    certifiedBy = certifiedBy && certifiedBy.replace(/\s+/g, ' ').trim()
+    this.emitValid(Boolean(certifiedBy && this.isCertified))
+    return certifiedBy
+  }
+
+  /** Emits an event to update the Is Certified prop. */
+  @Emit('update:isCertified')
+  private emitIsCertified (isCertified: boolean): boolean {
+    this.emitValid(Boolean(this.trimmedCertifiedBy && isCertified))
+    return isCertified
+  }
+
+  /** Emits an event indicating whether or not the form is valid. */
+  @Emit('valid')
+  private emitValid (valid: boolean): void { }
+}
+</script>
+
+<style lang="scss" scoped>
+// @import '@/assets/styles/theme.scss';
+
+#AR-step-4-container {
+  margin-top: 1rem;
+  padding-bottom: 0.5rem;
+  padding-top: 1rem;
```

Comment:
Similar to other components (such as StaffPayment), the component itself should not specify the whitespace around itself -- it should be as tight as possible -- then its parent can set the appropriate whitespace as needed.
~So please remove margin and padding from here.~

Reply:
The idea is sound but this is consistent with shared StaffPayment component.
Just leave this alone for now.